### DISCUSSION TITLE
Log cannot be collected after the log-rotate plugin is enabled

### DISCUSSION THREAD

**Discussion by lihuawei321** [apache/apisix#5657] [Q&A]
![image](https://user-images.githubusercontent.com/31987918/144166482-f843cff6-4a55-4f52-a1f8-f0a6d8579835.png)
![image](https://user-images.githubusercontent.com/31987918/144166507-a1140092-6715-42ba-8d7f-1eaf5693de28.png)
当我开启log-rotate插件后，发起请求后access.log没有日志，关掉log-rotate就有了

**Comment by tzssangglass** [comment]
work in process.

**Reply by moonming** [reply]
@tzssangglass is there has PR to fix?

**Reply by tzssangglass** [reply]
just fixed by: https://github.com/apache/apisix/pull/5715
